FeatureTest.java

```diff
@@ -6,7 +6,8 @@ public static void main(String[] args) {
     public int square(int variable){
         String str = null;
         System.out.println(str.length());
-        str = "fjff";
```

Comment:
Use a logger like SLF4J, log at ERROR level, and include exception details: logger.error("Null pointer exception occurred", e);